content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+| `DiskPressure`       | `True` if pressure exists on the disk size—that is, if the disk capacity is low; otherwise `False` |
+| `MemoryPressure`     | `True` if pressure exists on the node memory—that is, if the node memory is low; otherwise `False` |
+| `PIDPressure`        | `True` if pressure exists on the processes—that is, if there are too many processes on the node; otherwise `False` |
+| `NetworkUnavailable` | `True` if the network for the node is not correctly configured, otherwise `False` |
+{{< /table >}}
+
+{{< नोट >}}
+यदि आप कॉर्डन किए गए नोड के विवरण को प्रिंट करने के लिए कमांड-लाइन टूल का उपयोग करते हैं, तो शर्त में शामिल हैं
+'शेड्यूलिंग अक्षम'। कुबेरनेट्स एपीआई में `शेड्यूलिंग डिसेबल` एक शर्त नहीं है; बजाय,
+कॉर्डन किए गए नोड्स को उनकी कल्पना में अनिर्धारित चिह्नित किया गया है।
+{{< /नोट >}}
+
+Kubernetes API में, नोड की स्थिति को `.status` के भाग के रूप में दर्शाया जाता है
+नोड संसाधन का। उदाहरण के लिए, निम्न JSON संरचना एक स्वस्थ नोड का वर्णन करती है:
+
+```json
+"conditions": [
+  {
+    "type": "Ready",
+    "status": "True",
+    "reason": "KubeletReady",
+    "message": "kubelet is posting ready status",
+    "lastHeartbeatTime": "2019-06-05T18:38:35Z",
+    "lastTransitionTime": "2019-06-05T11:41:27Z"
+  }
+]
+```
+
+यदि तैयार स्थिति की `स्थिति` लंबे समय तक `अज्ञात` या `गलत` बनी रहती है
+`पॉड-एविक्शन-टाइमआउट` की तुलना में (एक तर्क पास किया गया
+{{< glosary_tooltip text="kube-controller-manager" term_id="kube-controller-manager"
+>}}), फिर [नोड नियंत्रक](#नोड-नियंत्रक) ट्रिगर करता है
+{{< glosary_tooltip text="API-initiated eviction" term_id="api-eviction" >}}
+उस नोड को सौंपे गए सभी पॉड्स के लिए। डिफ़ॉल्ट निष्कासन समयबाह्य अवधि है
+**पाँच मिनट**।
+कुछ मामलों में जब नोड पहुंच योग्य नहीं होता है, तो एपीआई सर्वर संवाद करने में असमर्थ होता है
+नोड पर क्यूबलेट के साथ। पॉड्स को हटाने के निर्णय के बारे में सूचित नहीं किया जा सकता है
+क्यूबलेट जब तक एपीआई सर्वर के साथ संचार फिर से स्थापित नहीं हो जाता। इस बीच में,
+विलोपन के लिए निर्धारित पॉड्स विभाजित नोड पर चलना जारी रख सकते हैं।
+
+नोड नियंत्रक पॉड को हटाने के लिए बाध्य नहीं करता है जब तक यह पुष्टि नहीं हो जाती कि वे बंद हो गए हैं
+क्लस्टर में चल रहा है। आप उन पॉड्स को देख सकते हैं जो अगम्य नोड पर चल रहे हैं
+`टर्मिनेटिंग` या `अज्ञात` स्थिति में होना। ऐसे मामलों में जहां कुबेरनेट्स से निष्कर्ष नहीं निकाला जा सकता है
+अंतर्निहित आधारभूत संरचना यदि कोई नोड स्थायी रूप से क्लस्टर छोड़ देता है, तो क्लस्टर व्यवस्थापक
+नोड ऑब्जेक्ट को हाथ से हटाने की आवश्यकता हो सकती है। कुबेरनेट्स से नोड वस्तु को हटाने का कारण बनता है
+नोड पर चल रहे सभी पॉड ऑब्जेक्ट्स को एपीआई सर्वर से हटा दिया जाएगा और उनके नाम को मुक्त कर दिया जाएगा।
+
+जब नोड्स पर समस्याएँ होती हैं, तो कुबेरनेट्स नियंत्रण विमान स्वचालित रूप से बनाता है
+[दाद](/docs/concepts/scheduling-eviction/taint-and-toleration/) जो शर्तों से मेल खाते हैं
+नोड को प्रभावित करना।
+नोड को पॉड असाइन करते समय शेड्यूलर नोड के दोष को ध्यान में रखता है।
+पॉड में {{< glosary_tooltip text="tolerations" term_id="toleration" >}} भी हो सकता है जो
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.